tests/wicked/aggregate/sut/t21_teaming_ifcfg_check_port_config.pm

```diff
@@ -0,0 +1,81 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: wicked teamd
+# Summary: Teaming, ifcfg - check port config
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use Mojo::Base 'wickedbase';
+use Mojo::JSON qw(decode_json);
+use Mojo::JSON::Pointer;
+use testapi;
+
+sub validate_team_port_config {
+    my ($self, $ctx) = @_;
+    my $iface1 = $ctx->iface();
+    my $iface2 = $ctx->iface2();
+
+    $self->wicked_command('ifup', 'team0');
+    $self->validate_interfaces('team0', $ctx->iface(), $ctx->iface2(), 0);
+    $self->ping_with_timeout(type => 'host', interface => 'team0', count_success => 30, timeout => 4);
+
+    my $jpointer = Mojo::JSON::Pointer->new(decode_json(script_output('teamdctl team0 config dump actual')));
+
+    my %check = (
+        '/device' => 'team0',
+        "/ports/$iface1/prio" => 10,
+        "/ports/$iface2/prio" => 1,
+        "/ports/$iface1/sticky" => undef,
+        "/ports/$iface2/sticky" => Mojo::JSON->true,
+    );
+    while (my ($k, $v) = each(%check)) {
+        my $val = $jpointer->get($k);
+        die("Teamd configuration failed in $k: got:'" . ($val // 'undef') . "' expected: '" . ($v // 'undef') . "'")
+          unless ((defined($val) && $val eq $v) || (!defined($val) && !defined($v)));
+    }
+    $self->wicked_command('ifdown', 'team0');
+}
+
+sub run {
+    my ($self, $ctx) = @_;
+    return if $self->skip_by_wicked_version('>=0.6.70');
+
+    record_info('INFO', 'Teaming, ifcfg - check port config');
+
+    my $ipaddr4 = $self->get_ip(type => 'host', netmask => 1);
+    my $iface1 = $ctx->iface();
+    my $iface2 = $ctx->iface2();
+
+    $self->write_cfg('/etc/sysconfig/network/ifcfg-team0', <<EOT);
```

Comment:
are sure that `);` should not be after second "EOT" ?

Reply:
another point - I know that you using this approach of keeping configs in the code in wlan tests but all aggregate tests are written with the flow where we keeping config files in "data" folder . Maybe it make sense to keep same flow in all tests ? at least in border of tests "family" ...

Reply:
> are sure that `);` should not be after second "EOT" ?

It's the here document of perl: https://en.wikipedia.org/wiki/Here_document#Perl . Ithink the syntax is correct.

Reply:
no I am not insisting